IdentityCore/src/util/MSIDTestIdTokenUtil.m

```diff
@@ -74,4 +80,17 @@ + (NSString *)idTokenWithName:(NSString *)name
     return [NSString stringWithFormat:@"%@.%@.%@", idTokenp1, idTokenp2, idTokenp1];
 }
 
++ (NSString *)idTokenWithName:(NSString *)name
+                          upn:(NSString *)upn
+                     tenantId:(NSString *)tid
+{
+    NSString *idTokenp1 = [@{ @"typ": @"JWT", @"alg": @"RS256", @"kid": @"_UgqXG_tMLduSJ1T8caHxU7cOtc"} msidBase64UrlJson];
```

Comment:
out of curiosity, what is a kid? it seems hashed, is this a valid id? in this case, should we replace it with something generic?

Reply:
Here's the documentation: https://tools.ietf.org/html/rfc7515#section-4.1.4
Maybe, yes, we might want to change it.